status-whisper-usage-spec.md

```diff
@@ -20,289 +36,145 @@ transport layer on top of Whisper.
-
-There is some tight coupling between the payload and Whisper:
-* Whisper message topic depends on the actual message type (see [Topic](#topic))
-* Whisper message uses a different key (asymmetric or symmetric) depending on the actual message type (see [Keys management](#keys-management))
+* *Offline message*: an archived envelope
+* *Envelope*: encrypted message with metadata like topic and Time-To-Live
 
 ## Whisper node configuration
 
-If you want to run a Whisper node and receive messages from Status clients, it must be properly cnofigured.
-
-Whisper's Proof Of Work algorithm is used to deter denial of service and various spam/flood attacks against the Whisper network. The sender of a message must perform some work which in this case means processing time. Because Status' main client is a mobile client, this easily leads to battery draining and poor performance of the app itself. Hence, all clients MUST use the following Whisper node settings:
+Whisper's Proof Of Work algorithm is used to deter denial of service and various spam/flood attacks against the Whisper network. The sender of a message MUST perform some work which in this case means processing time. Because Status' main client is a mobile client, this easily leads to battery draining and poor performance of the app itself. Hence, all clients MUST use the following Whisper node settings:
 * proof-of-work not larger than `0.002`
 * time-to-live not lower than `10` (in seconds)
 
-<!-- TODO: provide an instruction how to start a Whisper node with proper configuration using geth.-->
-
-<!-- @TODO: is there a higher bound -->
-
-## Keys management
-
-The protocol requires a key (symmetric or asymmetric) for the following actions:
-* signing a message (a private key)
-* decrypting received messages (a private key or symmetric key).
-
-As private keys and symmetric keys are required to process incoming messages, they must be available all the time and are stored in memory.
-
-Keys management for PFS is described in [Perfect forward secrecy section](#perfect-forward-secrecy-pfs).
-
-## Encryption algorithms
-
-All encryption algorithms used by Whisper should be described in the [Whisper V6 specification](http://eips.ethereum.org/EIPS/eip-627).
-
-Cryptographic algoritms used by PFS are described in [Perfect forward secrecy section](#perfect-forward-secrecy-pfs).
+## Topics
 
-### Topics
+The Status protocols uses a few particular Whisper topics to achieve its goals.
 
-There are two types of Whisper topics the protocol uses:
-* static topic for `user-message` message type (also called _contact discovery topic_)
-* dynamic topic based on a chat name for `public-group-user-message` message type.
+### Contact code topic
 
-The static topic is always the same and its hex representation is `0xf8946aac`.
-In fact, _the contact discovery topic_ is calculated using a dynamic topic
-algorithm described below with a constant name `contact-discovery`.
+Contact code topic is used for ???
```

Comment:
@cammellos I need some help with this one :)

Reply:
Contact code topic is used to facilitate the discovery of x3dh bundles, so the first message can be pfs encrypted.
Each user publishes periodically to this topic, if  I want to contact x, I will look for their bundle on this contact code topic.